nlp-utils/break-detection/src/main/resources/xml/break-and-reshape.xpl

```diff
@@ -138,7 +137,7 @@
     <p:with-option name="ensure-sentence-before" select="$ensure-sentence-before"/>
     <p:with-option name="ensure-sentence-after" select="$ensure-sentence-after"/>
   </px:break-detect>
-  <cx:message message="Java-based break detection done."/>
+  <px:message message="Java-based break detection done."/>
```

Comment:
When using `px:message` in utility modules we should not use the INFO severity (which is default). For these two messages you can add `severity="DEBUG"`.

See daisy/pipeline-issues#443 and daisy/pipeline-issues#442